frontend/src/api/APIWorker.tsx

```diff
@@ -1,83 +1,79 @@
 import { useEffect, useRef, useState } from 'react'
-import { showVesselsLastPosition } from '../domain/use_cases/vessel/showVesselsLastPosition'
-import { batch, useDispatch, useSelector } from 'react-redux'
-import getAllGearCodes from '../domain/use_cases/gearCode/getAllGearCodes'
-import { updateVesselTracks } from '../domain/use_cases/vessel/updateVesselTracks'
-import { setIsUpdatingVessels } from '../domain/shared_slices/Global'
-import { getAllFleetSegments } from '../domain/use_cases/fleetSegment/getAllFleetSegments'
-import getHealthcheck from '../domain/use_cases/healthcheck/getHealthcheck'
-import { getVesselVoyage } from '../domain/use_cases/vessel/getVesselVoyage'
-import { getVesselControls } from '../domain/use_cases/missions/getVesselControls'
+
 import { VesselSidebarTab } from '../domain/entities/vessel/vessel'
-import getAllRegulatoryLayers from '../domain/use_cases/layer/regulation/getAllRegulatoryLayers'
+import { setIsUpdatingVessels } from '../domain/shared_slices/Global'
 import { getOperationalAlerts } from '../domain/use_cases/alert/getOperationalAlerts'
+import { getSilencedAlerts } from '../domain/use_cases/alert/getSilencedAlerts'
 import getAllBeaconMalfunctions from '../domain/use_cases/beaconMalfunction/getAllBeaconMalfunctions'
-import { openBeaconMalfunctionInKanban } from '../domain/use_cases/beaconMalfunction/openBeaconMalfunctionInKanban'
 import getVesselBeaconMalfunctions from '../domain/use_cases/beaconMalfunction/getVesselBeaconMalfunctions'
-import getAllSpecies from '../domain/use_cases/species/getAllSpecies'
-import getVesselReportings from '../domain/use_cases/vessel/getVesselReportings'
-import { getSilencedAlerts } from '../domain/use_cases/alert/getSilencedAlerts'
-import getFishingInfractions from '../domain/use_cases/infraction/getFishingInfractions'
+import { openBeaconMalfunctionInKanban } from '../domain/use_cases/beaconMalfunction/openBeaconMalfunctionInKanban'
 import getAllControllers from '../domain/use_cases/controller/getAllControllers'
+import { getAllFleetSegments } from '../domain/use_cases/fleetSegment/getAllFleetSegments'
```

Comment:
To avoid these endless lists of imports, I find it's more readable to "index" all the actions in [this file](https://github.com/MTES-MCT/monitorfish/blob/master/frontend/src/domain/actions.ts).

It's not ideal to do that in general because I think (I need to check that) it doesn't allow the detection of unused code. We don't do it yet but will surely do it once all the files have been migrated to TS. However in the case of Redux actions, I think it's an acceptable tradeoff since there are easy to recheck manually.

It's also a nice way to start namespacing which shorten the name of Redux actions (both reducers and dispatcher) without having to repeat the name of the entity in te name of the function, i.e.:

`vesselActions.updateTracks()` instead of `updateVesselTracks()`